src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -79,9 +87,10 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
   const roomEvents = useMemo(() => {
     if (!venueEvents) return [];
 
-    return venueEvents.filter(
-      (event) =>
-        event.room === room.title && isBefore(Date.now(), eventEndTime(event))
+    return sortEventsByStartUtcSeconds(
+      venueEvents.filter(
+        (event) => event.room === room.title && isEventLiveOrFuture(event)
+      )
```

Comment:
if you apply the changes from my another comment (about RoomModal events), we won't need this filtering here,
so we'll need just

```js
const roomEvents = useMemo(() => {
    if (!venueEvents) return [];

    return sortEventsByStartUtcSeconds(venueEvents);
  }, [venueEvents]);
```

Reply:
fixed here: 31c51d873d13046cb48881bf72a6fe0f66072998